pkg/kubelet/kuberuntime/kuberuntime_manager_test.go

```diff
@@ -3070,8 +3072,8 @@ func TestToKubeContainerImageVolumes(t *testing.T) {
 		"empty volumes": {},
 		"multiple volumes": {
 			pullResults: imageVolumePulls{
-				volume1: imageVolumePullResult{spec: imageSpec1},
-				volume2: imageVolumePullResult{spec: imageSpec2},
+				volume1: imageVolumePullResult{spec: &imageSpec1},
```

Comment:
similar question about why a change from struct to pointer was required, and if that represents changes in the types that will break consumers as well